franklab_mountainsort/ms4_franklab_proc2py.py

```diff
@@ -377,7 +375,7 @@ def get_epoch_offsets(dataset_dir, opts=None):
     Parameters
     ----------
     dataset_dir : str
-    opts : None or dict, optional
+    opts : None or dict, optional*
```

Comment:
Not sure why there's an asterisk here? I assume typo?

Reply:
typo